.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and site, and run the deploy script.
+# The deploy script will publish the build to gh-pages.
+# This workflow will be run whenever a new release is published.
+
+# NOTE (pradeep): I'm having this workflow bail out if the release doesn't target v1.2.x. 
+# This is because we don't have a way (yet) to support multiple FDT versions/docs  be deployed at the same time,
+# and hence we choose to deploy only the most stable release.
+
+# Guide: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Deploy to gh-pages
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  check-release:
+    name: Check if release is stable
+    runs-on: ubuntu-latest
+
+    # map step output to job output
+    outputs: 
+      is_stable: ${{ steps.check-release-stable.outputs.is_stable }}
+
+    steps:
+    - name: Check if release is stable
+      id: check-release-stable
+      env:
+        # Store the tag name of the release in `RELEASE_TAG` (eg: v1.0.10)
+        RELEASE_TAG: ${{ github.event.release.tag_name }}
+
+      # Check if the release tag name is a stable candidate for deployment
+      # NOTE (pradeep): Only allow 1.2.x versions to be deployed right now
+      run: |
+        echo "RELEASE_TAG is: " $RELEASE_TAG
+        if [[ $RELEASE_TAG =~ ^v1\.2\.[0-9]+$ ]]; 
+        then
+          echo "This is a stable release!"
+          echo "::set-output name=is_stable::true"
+        else
+          echo "This is not a stable release!"
+          echo "::set-output name=is_stable::false"
+        fi
```

Comment:
In the future, we'll be customizing the deploy steps to support multiple releases.